packages/react/src/components/LocaleModal/LocaleModalRegions.js

```diff
@@ -71,11 +71,13 @@ const LocaleModalRegions = ({
           document.getElementById(`${prefix}--locale-modal__filter`).value = '';
           btn.removeAttribute('tabindex');
           btn.removeAttribute('role');
+          btn.removeAttribute('aria-label');
         };
 
         [...localeBackBtn].forEach(btn => {
           btn.setAttribute('tabindex', '1');
           btn.setAttribute('role', 'button');
+          btn.setAttribute('aria-label', 'back');
```

Comment:
@kennylam do you recall if there is a "Back" label we can leverage here instead of hardcoding the english only label here?

cc: @RaphaelAmadeu

Reply:
@jeffchew Yep you're right. That label is already provided by the translation file.

@RaphaelAmadeu `headerTitle` (Select region) is the one you want.

Reply:
@kennylam , Thanks, I'll change the label with the `headerTitle`